src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/03-create-cluster-create-services.e2e-spec.ts

```diff
@@ -39,7 +39,7 @@ describe('Create cluster create services page', () => {
     });
 
     it('should edit a service', () => {
-      const count = '3';
+      const count = '4';
```

Comment:
why's this?

Reply:
I am making sure all the host has rgw deployed so that when I force maintenance a host, the force maintenance action will behave exactly like it should. (since we are now deploying 4 hosts)

Reply:
Then should this be `hostCount` or something like that? Anyways, looks good.

Reply:
Technically it should be the no. of daemons after service is created and also its called as count in the service placement yaml. That's why I just went for count.

Reply:
The more intentional, the better: `daemonCount` sounds good to me.

Reply:
Thanks @pereman2 @alfonsomthd done!